src/ts/components/hugeplaybacktogglebutton.ts

```diff
@@ -124,6 +124,14 @@ export class HugePlaybackToggleButton extends PlaybackToggleButton {
     player.addEventHandler(player.EVENT.ON_CAST_START, castInitializationHandler);
     player.addEventHandler(player.EVENT.ON_CAST_STARTED, castInitializationHandler);
     player.addEventHandler(player.EVENT.ON_CAST_STOPPED, castInitializationHandler);
+
+    // Hide the play button animation when the UI is loaded (it should only be animated on state changes)
+    this.setTransitionAnimationsEnabled(false);
+
+    // Enable the transition animations when the play state changes for the first time
+    this.onToggle.subscribeOnce(() => {
+      this.setTransitionAnimationsEnabled(true);
```

Comment:
woulnd't it be more simple to add the fade animation class here instead of supressing it before with a different css class?

Reply:
It would save one line of code but wouldn't be as nicely backwards compatible. This solution can be easily merged into custom branches of customers because it only adds an additional CSS rule, your proposal would make it necessary to modify existing rules that would create merge conflicts.

Reply:
I have evaluated both approaches and the result is the same so I'm sticking with the current one.